docs/workflows/_bind-passkey-backend-return.mdx

```diff
@@ -0,0 +1,106 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Get Binding Link for Identity
+
+In order to get a binding link for a passkey, we need to create a binding link job for an exisiting identity. The following code snippet using the `RETURN` option. This is the fastest way to get a binding link as this method indicates that a binding link will be returned to the caller upon creation of the binding job. This binding link is the link you will send to your application to complete the passkey binding process.
```

Comment:
```suggestion
In order to get a binding link for a passkey, you need to create a binding job for an existing identity. The following code snippet uses the `RETURN` delivery method. This is the fastest way to get a binding link as this method indicates that a binding link will be returned to the caller upon creation of the binding job. This binding link is the link you will send to your application to complete the passkey binding process.
```

Should also check with the technical writers and product on this one. We should try to be consistent about which pronouns are used in the documentation (you vs we).